src/styles/vendor/react-mosaic-jss.js

```diff
@@ -0,0 +1,295 @@
+    },
+    '.mosaic-split.-row .mosaic-split-line': {
+      top: '0',
+      bottom: '0',
+      left: 3,
+      right: 3,
+    },
+    '.mosaic-split.-column': {
+      marginTop: -3,
+      height: 6,
+      cursor: 'ns-resize',
+    },
+    '.mosaic-split.-column .mosaic-split-line': {
+      top: 3,
+      bottom: 3,
+      left: '0',
+      right: '0',
+    },
+    '.mosaic-tile': {
+      position: 'absolute',
+      margin: 3,
+    },
+    '.mosaic-tile > *': {
+      height: '100%',
+      width: '100%',
+    },
+    '.mosaic-drop-target': {
+      position: 'relative',
+    },
+    '.mosaic-drop-target.drop-target-hover .drop-target-container': {
+      display: 'block',
+    },
+    '.mosaic-drop-target.mosaic > .drop-target-container .drop-target.left': {
+      right: 'calc(100% -  10px )',
+    },
+    '.mosaic-drop-target.mosaic > .drop-target-container .drop-target.right': {
+      left: 'calc(100% -  10px )',
+    },
+    '.mosaic-drop-target.mosaic > .drop-target-container .drop-target.bottom': {
+      top: 'calc(100% -  10px )',
+    },
+    '.mosaic-drop-target.mosaic > .drop-target-container .drop-target.top': {
+      bottom: 'calc(100% -  10px )',
+    },
+    '.mosaic-drop-target .drop-target-container': {
+      position: 'absolute',
+      top: '0',
+      right: '0',
+      bottom: '0',
+      left: '0',
+      display: 'none',
+    },
+    '.mosaic-drop-target .drop-target-container.-dragging': {
+      display: 'block',
+    },
+    '.mosaic-drop-target .drop-target-container .drop-target': {
+      position: 'absolute',
+      top: '0',
+      right: '0',
+      bottom: '0',
+      left: '0',
+      background: 'rgba(0, 0, 0, 0.2)',
+      border: '2px solid black',
+      opacity: '0',
+      zIndex: '5',
+    },
+    '.mosaic-drop-target .drop-target-container .drop-target.left': {
+      right: 'calc(100% -  30% )',
+    },
+    '.mosaic-drop-target .drop-target-container .drop-target.right': {
+      left: 'calc(100% -  30% )',
+    },
+    '.mosaic-drop-target .drop-target-container .drop-target.bottom': {
+      top: 'calc(100% -  30% )',
+    },
+    '.mosaic-drop-target .drop-target-container .drop-target.top': {
+      bottom: 'calc(100% -  30% )',
+    },
+    '.mosaic-drop-target .drop-target-container .drop-target.drop-target-hover': {
+      opacity: '1',
+    },
+    '.mosaic-drop-target .drop-target-container .drop-target.drop-target-hover.left': {
+      right: 'calc(100% -  50% )',
+    },
+    '.mosaic-drop-target .drop-target-container .drop-target.drop-target-hover.right': {
+      left: 'calc(100% -  50% )',
+    },
+    '.mosaic-drop-target .drop-target-container .drop-target.drop-target-hover.bottom': {
+      top: 'calc(100% -  50% )',
+    },
+    '.mosaic-drop-target .drop-target-container .drop-target.drop-target-hover.top': {
+      bottom: 'calc(100% -  50% )',
+    },
+    '.mosaic-window, .mosaic-preview': {
+      position: 'relative',
+      display: 'flex',
+      fallbacks: [
```

Comment:
this fallbacks array does not serialize with JSS.

Reply:
I've updated this in latest commit